ext/opentelemetry-ext-flask/tests/test_flask_integration.py

```diff
@@ -36,7 +36,9 @@ def setspanattr(key, value):
 
         self.span.set_attribute = setspanattr
 
-        self.app = Flask(__name__)
+        FlaskPatcher().patch()
```

Comment:
It may cause issues, but I think it is necessary for patching purposes. I can investigate a way that the patching is done in a way that only happens for the testing purposes of the patching agent. Also, I can think about a unpatching method that gets called automatically when patching is no longer needed.

Reply:
Does the `reload` function not work for your purposes? I believe it should reset everything you've patched: https://docs.python.org/3/library/importlib.html#importlib.reload

Although maybe a best-effort unpatch API is also a good idea as it'll be hard for people to be aware of the new patches that were added from version to version, so they may find their previous custom unpatching function not unpatching new patches added by a newer version of the auto-instrumentation.